### DISCUSSION TITLE
Timing/Grid Fault Issues SO 2.3.61

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#5033] [Unsupported Versions] [answered]
After building out my grid I noticed that the first search node wasn't properly set to UTC time. I tried manually setting the time to match SO manager but the grid still reports it in a fault state. I tried rebooting with no luck. The second search node was set to UTC time and joined to SO manager with no issues. How can clear the fault state and properly sync SO search node to SO manger?
![image](https://user-images.githubusercontent.com/70167373/127921868-ab4f405f-96fe-4548-a12e-d5e74b663adb.png)

**Comment by TOoSmOotH** [comment, marked as answer]
What do the logs say?

**Reply by ColeVan** [reply]
Timing was off. After removing from grid and setting UTC time in bios everything synced correctly
